brian2/tests/test_GSL.py

```diff
@@ -103,8 +103,8 @@ def test_GSL_default_function():
 
         assert max(trace_conventional.v[0]-trace_GSL.v[0]) < max_difference/mV, \
                 ('difference between conventional and GSL output is larger than max_difference')
-        assert not all(trace_conventional.v[0]==trace_GSL.v[0]), \
-                ('output of GSL stateupdater is exactly the same as Brians stateupdater (unlikely to be right)')
+        # assert not all(trace_conventional.v[0]==trace_GSL.v[0]), \
+        #         ('output of GSL stateupdater is exactly the same as Brians stateupdater (unlikely to be right)')
```

Comment:
Why is this commented out?

Reply:
Ah, right. The test was a bit weird, checking that the results of two state updaters for the same equations were *not* exactly the same. It was more a "during-development" test, Charlee used it to make sure that the simulation was actually using her state updater. I think this test failed with `float32`, i.e. the results were exactly the same. We can simply delete it.